src/dev/mod.ts

```diff
@@ -50,7 +53,10 @@ export interface Manifest {
 }
 
 const GROUP_REG = /[/\\\\]\((_[^/\\\\]+)\)[/\\\\]/;
-export async function collect(directory: string): Promise<Manifest> {
+export async function collect(
+  directory: string,
+  options: FreshOptions,
```

Comment:
Should the `options` here be optional? I went with non-optional, since the `dev` function also has it as non optional.

Reply:
Perhaps this parameter should be optional.